src/ritika/Mall.java

```diff
@@ -0,0 +1,93 @@
+/* 1. Create a class named as Mall, which contains below methods
+a. itemAddedToCart()
+b. customerName()
+c. paymentDoneUsing()
+	i. Visa Card 
+	ii. Cash
+d. displayCutomerInfo()
+e. needBill()
+f. haveCarryBag()
+g. decideSection()
+
+i.e. console should print: Harsh brought 12 items and payment done using visa card in section 1B
+
+Expectations:
+� If items are less than 10 and payment done using cash, then send customer to section 1A
+� If items are more than 10 and payment done using visa card, then send customer to section 1B
+� If items are more than 10 and payment done using cash, then send customer to section 1C
+� If items are less than 10 and payment done using visa card, then send customer to section 1D
+
+Hint: itemAddedToCart method have int parameter, paymentDoneUsing() method have String parameter, needBill() have Boolean parameter & haveCarryBag have Boolean parameter
+
+*/
+package ritika;
+
+class Mall {
+	int numberOfItems;
+	String customerName;
+	boolean billNeeded;
+	boolean carryBag;
+	String paymentMethod;
+	boolean flag;
+	
+	void customerName(String customerName) {
+		this.customerName = customerName;
+	}
+	void needBill(boolean billNeeded){
+		this.billNeeded = billNeeded;
+	}
+	void haveCarryBag(boolean carryBag){
+		this.carryBag = carryBag;
+	}
+	void displayCutomerInfo() {
+		System.out.println("Customer's Name is " +customerName);
+		if (billNeeded == true)
+			System.out.println(customerName + " needs Bill");
+		else if (billNeeded == false)
+			System.out.println(customerName + " does not need Bill");
+		if (carryBag == true)
+			System.out.println(customerName + " has Carry Bag");
+		else if (carryBag == false)
+			System.out.println(customerName + " does not have Carry Bag");
+	}
+	
+	void itemAddedToCart(int numberOfItems){
+		this.numberOfItems = numberOfItems;
+	}
+	
+	void paymentDoneUsingCardOrCash(String paymentMethod) {
+		if (paymentMethod.equals("Card"))
```

Comment:
instead of using true & false, you should use only String
this.paymentMethod = paymentMethod